bip-tap-proof-file.mediawiki

```diff
@@ -129,6 +128,19 @@ blob with the following format:
 * type: 10 (<code>challenge_witness</code>)
 ** value:
 *** [<code>...*byte</code>:<code>challenge_witness</code>]
+* type: 11 (<code>asset_genesis</code>) [version >= 1]
```

Comment:
So I think that we actually just will have this be v0, so starting from the "ideal" version.

Reply:
Removed version bump commit.